airflow/values.yaml

```diff
@@ -0,0 +1,191 @@
+# Default values for airflow-helm
+
+replicas:
+  web: 1
+  worker: 1
+  flower: 1
+  scheduler: 1
+
+labels:
+  node_selector_key: openstack-control-plane
+  node_selector_value: enabled
+
+images:
+  airflow: docker.io/eanylin/airflow-helm:1.8.1-1.6.2-0.8
```

Comment:
I created a newer image @ docker.io/eanylin/airflow-helm:1.8.1-1.6.2-0.9

Hopefully this will be the final one.  Testing looks positive so far.